### ISSUE TITLE
eclipse err:  connot connect to dfs

### ISSUE THREAD

**Issue by renzhengeek** [winghc/hadoop2x-eclipse-plugin#8]
after installled plugin,and start eclipse, then to use hdfs, occure err:

An internal error occurred during: "Connecting to DFS MasterNode".
java.lang.reflect.InvocationTargetException
help~

**Comment by renzhengeek** [comment]
Maybe this error is related with verison of eclipse, please give me which version with witch you devolepe this 
plugin.thank!

**Comment by renzhengeek** [comment]
my enviroment is hadoop-2.2.0, ubuntu 12.04 32bit, eclipse kepler.

And I also tryed the hadoop-eclipse-kepler-plugin-2.2.0.jar.the err still the same.
I wonder wether my configuration of "Define the location of a Hadoop".
below is my configuration:
location name: MasterNode
DFS Master:     localhost:8020
MapReduce Master:   localhost:9001
username:renzhengeek
is this right?thanks!

**Comment by winghc** [comment]
That jar is for x64

Date: Fri, 10 Jan 2014 04:36:30 -0800

**Comment by renzhengeek** [comment]
oh thanks a lot.
and, how to make a one for x32?

**Comment by winghc** [comment]
It's better for you to compile the source code against  real environment.
Date: Mon, 13 Jan 2014 17:11:58 -0800

**Comment by renzhengeek** [comment]
yes, I tryed compiling the source code against  real environment. 
but still with the same problem.
